Common/Extensions/Course+NSUserActivity.swift

```diff
@@ -9,7 +9,7 @@ extension Course {
 
     public var openCourseUserActivity: NSUserActivity {
         let userActivity = NSUserActivity(activityType: Bundle.main.activityTypeOpenCourse.require())
-        userActivity.title = title
+        userActivity.title = url?.absoluteString
```

Comment:
Is this still required or was the issue resolved by using `NSItemProvider(object: self)`?
If we can use the course title here then I would prefer that.